addons/iap/models/iap_account.py

```diff
@@ -18,17 +18,101 @@ class IapAccount(models.Model):
     _rec_name = 'service_name'
     _description = 'IAP Account'
 
-    service_name = fields.Char()
-    account_token = fields.Char(default=lambda s: uuid.uuid4().hex)
+    service_name = fields.Char(string='Account Name')
+    account_token = fields.Char(default=lambda s: uuid.uuid4().hex,
+                                help="Account token is your authentication key for this service. Do not share it.",
+                                size=43)
     company_ids = fields.Many2many('res.company')
+    account_info_id = fields.Many2one(
+        'iap.account.info', compute='_compute_info', inverse='_inverse_info', search='_search_info')
+    account_info_ids = fields.One2many('iap.account.info', 'account_id',
+                                  invisible=True, string="Accounts from IAP")
+    iap_service_name = fields.Char(string='Related Service', related='account_info_id.service_name')
+    balance = fields.Char(compute='_compute_balance')
+    description = fields.Char(related='account_info_id.description')
+    warn_me = fields.Boolean(related='account_info_id.warn_me', readonly=False,
+                             help="We will send you an email when your balance gets below that threshold")
```

Comment:
```suggestion
    warn_me = fields.Boolean(
        related='account_info_id.warn_me',
        help="We will send you an email when your balance gets below that threshold")
```
It's not necessary to set `readonly=False`, it's already the default value.